modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/IntrospectionUtils.java

```diff
@@ -156,7 +158,14 @@ public static DataType getMethodArgumentDataType(Method method, int argumentInde
             return ImmutableMap.of();
         }
 
-        ImmutableMap.Builder<Method, DataType> map = ImmutableMap.builder();
+        ImmutableMap.Builder<Method, DataType> map = ImmutableSortedMap.orderedBy(new Comparator<Method>()
```

Comment:
this change will likely affect the generated schema. Why am I not seeing a change on schema generator test ?